src/community/design-system-day-2022/index.md

```diff
@@ -19,7 +19,7 @@ For the past 6 years, Amy has worked on design systems for some of the biggest n
 
 Whether she’s leading systems work or designing content, Amy works to champion simplicity and inclusion in the face of complexity.
 
-<iframe width="560" height="315" src="https://www.youtube.com/embed/25XuvRqbLEM" title="YouTube video player" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+<a href='https://www.youtube.com/watch?v=25XuvRqbLEM'>Watch 'Building conscious design systems' on YouTube</a>.
```

Comment:
```suggestion
<a href="https://www.youtube.com/watch?v=25XuvRqbLEM">Watch 'Building conscious design systems' on YouTube</a>.
```

Reply:
I used double quotes the other day and it broke the hyperlink. Maybe because it was inside a partial? Thanks for this, have tested and it doesn't break the link.